distributed/scheduler.py

```diff
@@ -4015,31 +4164,136 @@ def __init__(
         self.rpc.allow_offload = False
         self.status = Status.undefined
 
+    ####################
+    # state properties #
+    ####################
+
+    @property
+    def aliases(self):
+        return self.state.aliases
```

Comment:
Should `aliases` and `host_info` not rather be server attributes than state attributes?